vuepress/docs/next/tutorials/devops/installation/open-shift/openshift-install-by-operator-hub.md

```diff
@@ -262,6 +262,30 @@ Now we'll create the application itself.
 
 See the [Next Steps](#next-steps) below to continue your work with Entando.
 
+## Appendix A - Troubleshooting and Common Errors
+
+### SSL Certificate Error
+When installing Entando V6.3.2 into OpenShift 4.6, you may run into a SSL certificate error. This happens because of restricted registries for the docker image. 
```

Comment:
Personal preference maybe, but no comma?

Reply:
which comma?

Reply:
Sorry - just the one here: "...OpenShift 4.6**,** you..." I probably wouldn't use a comma in such a short sentence - but also don't think doing so is incorrect! Notwithstanding where it's technically required, my rule of thumb is to add a comma where I'd pause if i were reading something out loud. Thoughts?

Reply:
I think because of the introductory clause at the beginning, you need the comma.